pkg/kube/inject/inject.go

```diff
@@ -159,6 +162,7 @@ type Config struct {
 
 const (
 	SidecarTemplateName = "sidecar"
+	ZtunnelTemplateName = "ztunnel"
```

Comment:
```suggestion
	ZtunnelTemplateName = "ztunnel"
```

nit: do we need to call it `ztunnel`? it strictly speaking doesn't have anything to do with ztunnel, right?

`ambient` or `cni-wait` or `ambient-init`

Reply:
EDIT: Nevermind. I see, it is waiting for the ztunnel sockets.

I still think the name is overly inside-baseball tho, in general I don't wanna proliferate `ztunnel` in our explicit or implicit APIs. `Proxy`, `node proxy` or `ambient` or `CNI`.